census/src/test/scala/grasshopper/census/util/Generators.scala

```diff
@@ -0,0 +1,36 @@
+package grasshopper.census.util
+
+import org.scalacheck.{ Arbitrary, Gen }
+
+trait Generators {
+
+  case class Alphanumeric(s: String) {
+    override def toString = s.toString
+  }
+
+  case class HyphenStr(s: String) {
+    override def toString = s.toString
+  }
+
+  val unicodeChar = Gen.choose(Char.MinValue, Char.MaxValue).filter(c => !Character.isDigit(c))
+
+  val alphanumericGen: Gen[Alphanumeric] = {
+    for {
+      x <- Gen.oneOf("A", "B", "C")
+      y <- Gen.choose(Int.MinValue, Int.MaxValue)
+    } yield Alphanumeric(List(x, y).mkString)
+  }
+
+  implicit def arbAlphanumeric: Arbitrary[Alphanumeric] = Arbitrary(alphanumericGen)
+
+  val hyphenStrGen: Gen[HyphenStr] = {
+    for {
```

Comment:
Similar to above, do you want the range to start from `Int.MinValue`, which could result in negative numbers.  You could end up with something like `-123--456`, which doesn't seem like what we really want to test.  Seems like the range should start from 0 or 1.